zipkin_opentracing/src/opentracing.cc

```diff
@@ -246,8 +263,10 @@ class OtSpan : public ot::Span {
     return span_context_.baggageItem(restricted_key);
   }
 
-  void Log(std::initializer_list<std::pair<string_view, Value>>
-               fields) noexcept override {}
+  void Log(std::initializer_list<std::pair<string_view, Value>> fields) noexcept override {
+    std::lock_guard<std::mutex> lock_guard{mutex_};
+    log_records_.push_back({ SystemClock::now(), { fields.begin(), fields.end() } });
+  }
```

Comment:
This can potentially throw `std::bad_alloc`, so you need a try/catch block here like what's done in `SetTag`.